src/main/webapp/assets/css/global-custom.css

```diff
@@ -561,3 +561,7 @@ body{
 .stats {
     font-size: 13px;
 }
+
+.carousel-item{
```

Comment:
A small problem occurs on the mobile view with this. Can you please look into that also
![Screenshot_2020-06-22-19-20-07-004_com android chrome](https://user-images.githubusercontent.com/27498587/85296444-068eda80-b4bf-11ea-876d-e293a757d3d5.jpg)

Reply:
Fixed! @Piumal1999